### PR TITLE
Feature/blueprint template

### PR DESCRIPTION
**Pull request by hmunfru**
@jesuspg

### CODE REVIEW

**Comment by flopezag** [comment]
from feature/xxxx to master?

**Comment by flopezag** [comment]
Two comments:
- Introduce to @pratid and Javier Fernández in the reviewers of this code...
- Put print is not allowed.
- Apply the coding style of OpenStack (not only pep8 but also the other components).

**Comment by hmunfru** [comment]
murano works with master no with develop branch
OK about the reviewers
Print (my fault :-()
Executed by using tox

File: murano/utils.py [hunk 1]
```diff
@@ -46,6 +46,26 @@ def __inner(self, request, environment_id, *args, **kwargs):
     return __inner
 
 
+def verify_temp(func):
```
Comment (jesuspg) [inline murano/utils.py:49]: I recommend rename of method by verify_template

**Pull request closed without merging**